### PR TITLE
ColumnsLayout Documentation

### PR DESCRIPTION
**Pull request by bbrennanbasis**
This pull request if for the ColumnsLayout page.

### CODE REVIEW

File: docs/components/columns-layout.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+You can add a `List` of breakpoints during the construction of a `ColumnsLayout` component, or afterward with the `setBreakpoints` method.
+
+There are three parts to a breakpoint:
+
+1. **Name (optional)**:
+Naming a breakpoint enables you to reference it in future configurations, thereby altering the layout of a specific component exclusively at this breakpoint.
+
+2. **Minimum width**:
+This value establishes the starting point of a range that encompasses a breakpoint's behavior.
+When given an `int` value, this can be in pixels, or a `String` for other units like vw, %, or em.
```
Comment (gosteenBASIS) [inline docs/components/columns-layout.md:43]: I found this portion (2. Minimum width) didn't flow very well. I took a stab at a rewrite, but feel free to re-re-write it:

This value defines the minimum width for which this layout will take effect. The layout will apply to all sizes greater than the minimum width unless another `Breakpoint` takes effect. 
`int` values have the default unit of pixels, but you can use a `String` to supply any other standard units such as vw, %, or em.

File: docs/components/columns-layout.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+  new Breakpoint(0,1)
+  new Breakpoint(500,3)
+);
+
+ColumnsLayout columnsLayout = new ColumnsLayout(breaks, componentOne, componentTwo, componentThree);
+```
+
+## Column Spans
+
+A span is the horizontal section of one column. Changing the span for a component is telling it to take up multiple sections in a column layout.
```
Comment (gosteenBASIS) [inline docs/components/columns-layout.md:61]: Suggested rewrite for clarity:
A span is the horizontal section of one column. Changing the span for a component is telling it to take up multiple sections in a column layout.

A span represents the horizontal width of a column in the layout. By changing the span for a component, you can make it extend across multiple columns.

File: docs/components/columns-layout.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+List<Breakpoint> breaks = List.of(
+  new Breakpoint(0,1)
+  new Breakpoint(500,3)
+);
+
+ColumnsLayout columnsLayout = new ColumnsLayout(breaks, componentOne, componentTwo);
+columnsLayout.setSpan(componentTwo, 2);
+```
+
+Spans can be set for specific breakpoints by refrencing the breakpoint's name in the `setSpan` method.
```
Comment (gosteenBASIS) [inline docs/components/columns-layout.md:75]: Suggested rewrite (making "breakpoints" singular to match the number of "breakpoint's name") and typo fix (refrencing):
Spans can be set for specific breakpoints by refrencing the breakpoint's name in the `setSpan` method.

A span can be set for a specific breakpoint by referencing the breakpoint's name in the `setSpan` method, as in the example below:

**Comment by bbrennanbasis** [comment]
See Also: PR from the webforj-docs-samples archive: https://github.com/webforj/webforj-docs-samples/pull/52

**Comment by bbrennanbasis** [comment]
This pull request is absolute now. See PR https://github.com/webforj/webforj-docs/pull/233.

**Pull request closed without merging**